packages/htm/src/render.test.ts

```diff
@@ -3,17 +3,19 @@ import {
   createRemoteComponent,
   RemoteComponent,
 } from '@remote-ui/core';
-import {htm, render} from './render';
+import {createRender, append} from './render';
 
 const Button = createRemoteComponent('Button');
 
 describe('htm', () => {
   it('converts htm into RemoteRoot operations', () => {
     const root = createRemoteRoot(() => {});
+    const ui = createRender(root);
+
     const onPress = jest.fn();
-    const content = 'Buy now';
+    const content = 'Buy me!';
 
-    render(htm`<${Button} onPress=${onPress}>${content}<//>`, root);
+    append(ui`<${Button} onPress=${onPress}>${content}<//>`, root);
```

Comment:
Can we add another test to verify that using `append` on a non-root component works?